src/cli/run_command.zig

```diff
@@ -348,13 +349,23 @@ pub const RunCommand = struct {
             Output.flush();
         }
 
+        const envp = try env.map.createNullDelimitedEnvMap(allocator);
+        defer {
+            for (envp) |e| {
+                if (e) |ptr| {
+                    allocator.free(bun.span(ptr));
+                }
+            }
+            allocator.free(envp);
+        }
+
         const spawn_result = switch ((bun.spawnSync(&.{
             .argv = &argv,
             .argv0 = shell_bin.ptr,
 
             // TODO: remember to free this when we add --filter or --concurrent
```

Comment:
```suggestion
            // TODO: remember to free this when we add --concurrent
```

Reply:
envp is freed. this entire comment can be removed now.